src/app/trial-card/trial-card.component.html

```diff
@@ -1,22 +1,24 @@
 <!-- At some point the containing icon will likely be updated to include a class based on the actual status -->
-<div class="overall-status"><div>{{clinicalTrial.overallStatus}}</div></div>
+<div class="overall-status">
+  <div [ngStyle]="{'background': trialStatusColor(clinicalTrial.overallStatus)}">{{statusDisplay[clinicalTrial.overallStatus]}}</div>
```

Comment:
Rather than expose the display names directly from `fhir-constants` it would probably be better to add a method to the component class to include a check to ensure that the overall status actually exists within the status display. The types in `fhir-constants` includes the valid types but there's nothing that ensures that the data returned from the endpoint is actually valid FHIR.

Reply:
Added helper function to access FHIR-constants